src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Utility.cs

```diff
@@ -29,8 +29,8 @@ public static string EncryptUsingGatwewaySettings(GatewayStatus status, string s
             var result = string.Empty;
             var jwk = status.EncryptionJwk;
 
-#if PORTABLE
-            using (var rsa = RSA.Create())
+#if NET45
```

Comment:
switched to using above pattern for preprocessor directive
updated to latest swagger spec to include support for new installerDownload property and related test